templates/components/randomizer.html

```diff
@@ -0,0 +1,17 @@
+<style type="text/css">
+  .icon {
+    display: inline-flex;
+    place-items: center;
+    /* override picocss default text color style for `<a>` tag */
+    color: unset;
+    width: 20px;
+    height: 20px;
+  }
+</style>
+
+<a href="/random" class="icon">
```

Comment:
Can you add a comment to describe how this svg is generated? Also, curious why we can't use a simple emoji 🔀 for it (same as in `index.html`)?

Reply:
Emoji is very different  among different browsers while SVG not. I prefer to SVG.🤔

Reply:
Yes, will do. The SVG here matches the style of the existing icon:

![image](https://github.com/laike9m/Python-Type-Challenges/assets/12967000/eec9c734-84f9-4328-b23b-dbe3f8560393)

Reply:
Updated with https://github.com/laike9m/Python-Type-Challenges/pull/53/commits/23f4364d09d0e43a34a7365298cd3cf58fb73dfc

- Add tooltip
- Update button text to "Random Challenge"
- Move the button to the bottom of the page
- Add comment for the source of the SVG

Reply:
Thanks. Can we also add a tooltip on the randomizer icon on the challenge page?